src/start/DevAuthScreen.js

```diff
@@ -79,11 +79,11 @@ class DevAuthScreen extends PureComponent<Props, State> {
         <View style={styles.container}>
           {progress && <ActivityIndicator />}
           {error && <ErrorMsg error={error} />}
-          <Text style={[styles.field, styles.heading2, inlineStyles.heading]}>Administrators</Text>
+          <Label style={[styles.field, styles.heading2, inlineStyles.heading]}>Administrators</Label>
```

Comment:
We are using `Label` as `<Label style={styles.text} text="No Internet connection" />`
providing text in `text` props.

Reply:
Also, any string that is added to `Label` is being translated, so don't forget to add these to `messages_en.json`